girder/utility/webroot.py

```diff
@@ -90,7 +90,7 @@ def __init__(self, templatePath=None):
             'plugins': [],
             'apiRoot': '',
             'staticRoot': '',
-            'title': 'Girder',
+            'title': ModelImporter.model('setting').get(SettingKey.BRAND_NAME),
```

Comment:
And for compatibility, continue to pass `title` as a hardcoded value, with a code comment that it's depreciated. The `title` should no longer be used in the `webroot.mako` template though.